src/physics/initial_conditions/initial_condition_function.h

```diff
@@ -272,6 +272,50 @@ class InitialConditionFunction_1DSine
     real value (const dealii::Point<dim,real> &point, const unsigned int istate = 0) const override;
 };
 
+/// Initial Condition Function: Isentropic vortex
+template <int dim, int nstate, typename real>
+class InitialConditionFunction_IsentropicVortex
+        : public InitialConditionFunction<dim,nstate,real>
+{
+protected:
+    using dealii::Function<dim,real>::value; ///< dealii::Function we are templating on
+
+public:
+    /// Constructor for InitialConditionFunction_IsentropicVortex
+    /// Setup according to "A Survey of the Isentropic Euler Vortex Problem using High-Order Methods"
+    ///     Spiegel et al., 2015
+    /// Using "Shu" variant (first row of Table 1)
+    /// Non-dimensional initialization
+    /// Increased domain from L=5 -> L=10 per recommendation of Spiegel et al
```

Comment:
could use multiline comment like this ; applies for the KHI as well
<img width="814" alt="image" src="https://user-images.githubusercontent.com/43619715/222925155-83036286-6215-45c3-a0c9-2abc303f960c.png">

Reply:
Fixed